frontend/router/index.js

```diff
@@ -0,0 +1,26 @@
+import { createRouter, createWebHashHistory } from "vue-router";
+import Home from "../views/Home.vue";
```

Comment:
Maybe use lazy loading?
```
const Home = () => import("../views/Home.vue");
```

Also maybe use path aliases instead of this `../`
https://webpack.js.org/configuration/resolve/#resolvealias
As i can see, there is `@ is an alias to /src`

Reply:
Okay, I haven't checked most of these files generated by Vue CLI because a lot of them will be removed/changed so I left them to have some structure in files. But I will implement lazy loading for now